discovery-provider/src/api/v1/models/activities.py

```diff
@@ -55,3 +55,15 @@ def format(self, value):
         "item": FullActivityItem,
     },
 )
+
+track_activity_model_full = ns.clone(
+    "track_activity_full",
+    activity_model_full,
+    {"item_type": fields.Nested(track_full)},
```

Comment:
Wait, this isn't item type, this is the full object. I think you meant to have both `item_type` and `item`?

Reply:
omg. good catch